homeassistant/components/plex/server.py

```diff
@@ -37,18 +40,16 @@
 class PlexServer:
     """Manages a single Plex server connection."""
 
-    def __init__(self, server_config, options=None):
+    def __init__(self, hass, server_config, options=None):
         """Initialize a Plex server instance."""
+        self._hass = hass
         self._plex_server = None
-        self._clients = {}
-        self._sessions = {}
+        self._known_clients = []
```

Comment:
If all items in this list should be unique, make it a set instead. Sets have benefits when checking membership. You can add items without risking duplication, and check intersection, difference etc between sets and other sets or sets and dicts.
```suggestion
        self._known_clients = set()
```